src/Spryker/Client/RabbitMq/Model/Connection/ConnectionBuilder/ConnectionBuilder.php

```diff
@@ -114,14 +114,35 @@ protected function createAmqpStreamConnection(QueueConnectionTransfer $queueConn
             $queueConnectionTransfer->getLocale() ?? $this->getDefaultLocale(),
             $queueConnectionTransfer->getConnectionTimeout() ?? $defaultQueueConnectionTransfer->getConnectionTimeout(),
             $queueConnectionTransfer->getReadWriteTimeout() ?? $defaultQueueConnectionTransfer->getReadWriteTimeout(),
-            null,
+            $this->createSslContext($queueConnectionTransfer),
```

Comment:
I would expect to have more generic context here. Config value doesn't say anything that it's `ssl` only related

Reply:
True, I'll rename it.